assets/quick-start-module.js

```diff
@@ -519,79 +522,79 @@ function commandMessage(key) {
 
     "preview,pip,macos,cudanone,python3.5":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
 
     "preview,pip,macos,cuda9.2,python3.6":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,cuda10.0,python3.6":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,cudanone,python3.6":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
 
     "preview,pip,macos,cuda9.2,python3.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,cuda10.0,python3.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,cudanone,python3.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
 
     "preview,pip,linux,cudanone,python2.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch==" + torch_nightly_ver + "+cpu torchvision==" + vision_nightly_ver + "+cpu -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
```

Comment:
You don't need to hardcode the versions here

Reply:
I tried not to hardcode the version during installation but without success.
Log:
```
$>pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Looking in links: https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Collecting torch
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/f8/02/880b468bd382dc79896eaecbeb8ce95e9c4b99a24902874a2cef0b562cea/torch-0.1.2.post2.tar.gz (128kB)
     |████████████████████████████████| 133kB 652kB/s
ERROR: Operation cancelled by user
```

Reply:
Detailed log: https://gist.github.com/peterjc123/ae2e753bdd5628b62ad71984533a9584

Reply:
I think the fix is, you also need to pass `--pre` so pip considers preinstall versions too (I don't know if the mirror is involved with the trouble)

Reply:
Yes, it works. Thanks!